com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -675,20 +800,20 @@ half3 SubtractDirectMainLightFromLightmap(Light mainLight, half3 normalWS, half3
 }
 
 half3 GlobalIllumination(BRDFData brdfData, BRDFData brdfDataClearCoat, float clearCoatMask,
-    half3 bakedGI, half occlusion,
+    half3 bakedGI, half occlusion, half3 positionWS,
```

Comment:
Mention how the addition of posistionWS in functions can lead to breaking users code

Reply:
No longer the case with backward compatible overrides